client/src/app/components/Header.tsx

```diff
@@ -114,6 +117,7 @@ class Header extends Component<HeaderProps, HeaderState> {
                                                 to={utility.url}
                                                 onClick={() =>
                                                     this.setState({ isUtilitiesExpanded: false })}
+                                                className={utility.url === window.location.pathname ? "active" : ""}
```

Comment:
Así nos evitamos las comillas vacías
```suggestion
 className={classNames(
     { active: utility.url === window.location.pathname }
 )}
```